### DISCUSSION TITLE
Security & Dependabot notifications in app

### DISCUSSION THREAD

**Discussion by mobilemind** [community/community#18303] [Mobile]
Sure would be nice to see Security or Dependabot alerts for my app when I use the GitHub mobile app.

Thanks!!

**Comment by eliperkins** [comment]
Hey @mobilemind! I hear you on this one, getting security advisories and Dependabot alerts in GitHub Mobile sounds like an excellent feature that I'd love to use too.

I've shared your feedback with the team in an internal tracking issue for security advisory and Dependabot alerts. I'll keep you posted on how our team moves forward with this.

**Reply by DavNaty** [reply]
Ano
